tools/mason/mason.chpl

```diff
@@ -116,10 +117,28 @@ proc masonClean() {
   runCommand('rm -rf target');
 }
 
+ 
+proc masonDoc(args) {
+  var toDoc = basename(getEnv('PWD'));
+  var project = toDoc + '.chpl';
+  if isDir('src/') {
+    if isFile('src/' + project) {
+      var command = 'chpldoc src/' + project;
```

Comment:
Note: `mason` will now depend on `chpldoc`, so this dependency should be marked in the top-level `Makefile`